mypy/subtypes.py

```diff
@@ -560,6 +560,7 @@ def find_node_type(node: Union[Var, FuncBase], itype: Instance, subtype: Type) -
         else:
             typ = signature
     itype = map_instance_to_supertype(itype, node.info)
+    assert typ is not None
```

Comment:
This seems to be redundant, could you please add a comment why this is needed?

Reply:
It's a knock-on effect of Var's `__init__` getting typechecked in the second pass I think. But it is pretty bogus and probably a mypy bug, but I don't think I have the cycles right now to track it down. Annotating the FakeInfos fixes the problem, though. SIgh.